userland/examples/nrf51_aes/main.c

```diff
@@ -0,0 +1,204 @@
+#include <stdio.h>
```

Comment:
Well, it depends on how the initial counter is configured on other chips e.g. SAM4L.
Otherwise it should not be nrf51-specific.
If it's configured uniformly it should be no problem @alevy do you know? ;)